sdk/eventhub/Azure.Messaging.EventHubs/assets/live-tests-azure-setup.ps1

```diff
@@ -7,12 +7,12 @@
     library test suite.
 
   .DESCRIPTION
-    This script handles creation and configuration of needed resources within an Azure subscription
+    This script handles the creation and configuration of needed resources within an Azure subscription
     for use with the Event Hubs client library's Live test suite.  
     
     Upon completion, the script will output a set of environment variables with sensitive information which
-    are used for testing.  When running Live tests, please be sure to have these environment variables available,
-    either within Visual Studio or command line environment.
+    is used for testing.  When running Live tests, please be sure to have these environment variables available,

```

Comment:
nit:  The subject of the sentence is the "environment variables" which contain the information - not the information itself.  I believe `are` is correct in this case.